### ISSUE TITLE
string with mutation sometimes contains break line

### ISSUE THREAD

**Issue by ligurio** [gpetiot/Frama-C-Mutation#2]
I'm using resulted summary.csv to obtain information about mutated files, number of lines and mutated operators. Unfortunately sometimes strings with mutations contains break lines and it breaks CSV parsing.

Example:
```
rend1_7.c,rend1.c:5718: `msgOut->immSize > sizeof(msgOut->__anonCompField4.imm)` --> `! (
msgOut->immSize > sizeof(msgOut->__anonCompField4.imm))`
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
rend1_8.c,rend1.c:5718: `==` --> `!=`
rend1_9.c,rend1.c:5718: `msgOut->immSize == (rtl_uint32_t)0` --> `! (msgOut->immSize == (rtl_uint32_t)0)`
rend1_10.c,rend1.c:5713: `!=` --> `==`
```

latest version (43ca0f951c16660ea9791c55402ed4716f8baa44)

**Comment by gpetiot** [comment]
Could you test PR #4 and tell me if it is what you expect?

**Comment by ligurio** [comment]
I'll try, thanks!

**Comment by gpetiot** [comment]
Closed with PR #4
